admin-auth/src/test/kotlin/com/epam/drill/admin/auth/HexSecretGeneratorTest.kt

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.admin.auth
+
+import com.epam.drill.admin.auth.service.impl.HexSecretGenerator
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+/**
+ * Testing [HexSecretGenerator]
+ */
+class HexSecretGeneratorTest {
```

Comment:
Do we really need secret generator?
I don't see a realistic use case for that: restarting admin will wipe the secret and invalidate all keys, won't it?

Reply:
We attach `id` to each API key and _not_ using a instance-wide salt for all keys
So that is not the case